linera-rpc/src/rpc.rs

```diff
@@ -27,27 +27,60 @@ pub enum RpcMessage {
     Error(Box<NodeError>),
     // Internal to a validator
     CrossChainRequest(Box<CrossChainRequest>),
+    VersionInfoQuery,
+    VersionInfoResponse(Box<VersionInfo>),
```

Comment:
I was trying to have 3 sections: inbound / outbound / internal. If we don't want that remove the comments.